frontend/src/shared/ui/button/button.module.scss

```diff
@@ -4,50 +4,49 @@
   transition: all var(--fast--transition-delay);
 
   border: none;
-}
 
-.button:not(:disabled) {
-  cursor: pointer;
+  &:not(:disabled) {
+    cursor: pointer;
+  }
 }
 
 .button--primary {
   background-color: var(--primary-color);
   color: var(--white-color);
-}
 
-.button--primary:hover {
-  background-color: var(--primary-hover-color);
+  &:hover {
+    background-color: var(--primary-hover-color);
+  }
+  &:active {
+    background-color: var(--primary-active-color);
+  }
 }
 
-.button--primary:active {
-  background-color: var(--primary-active-color);
-}
-
-.button--primary:disabled, .button--secondary:disabled {
+.button--primary:disabled,
+.button--secondary:disabled {
   background-color: var(--disabled-bg-color);
   color: var(--disabled-font-color);
 }
```

Comment:
There are a couple of places with such style code. Decided not to refactor for better readability

Reply:
Uh, didn't see that. It can be refactored, its read as well.